### DISCUSSION TITLE
How can I add Theme switcher

### DISCUSSION THREAD

**Discussion by kanhaiya5555** [webstudio-is/webstudio-community#140] [Q&A]
Hi my name is Kanhaiya I am a designer.
How do I switch from dark theme to light theme in WebStudio? With a single click.

**Comment by kof** [comment]
Do you mean how to create a theme switcher on your site? or are you looking for a way to use webstudio builder with dark mode?

**Comment by 0ymane** [comment]
@kof yes indeed i still haven't figured out how to use or enable dark mode.
plus i am used to change the css in the custom code panel in webflow that target the variables and use data attributes and that sort of stuff, so i am wondering if there is possibility to achieve the same with the tokens ?

**Reply by kof** [reply]
you can use attributes just the same way

**Reply by kof** [reply]
in the nexts release we will add css variables, so that will be the preferred way to do theming.
